features/environment.py

```diff
@@ -149,21 +151,29 @@ def create_trusty_uat_lxd_image(context: Context) -> None:
     :param context:
         A `behave.runner.Context`; this will have `image_name` set on it.
     """
-    now = datetime.datetime.now()
-    context.image_name = "behave-image-" + now.strftime("%s%f")
-    build_container_name = "behave-image-build-" + now.strftime("%s%f")
+    if context.container_to_run:
+        print(
+            "It is going to run the test in the existent container: ",
+            context.container_to_run,
+        )
+    else:
+
+        now = datetime.datetime.now()
+        context.image_name = "behave-image-" + now.strftime("%s%f")
+        build_container_name = "behave-image-build-" + now.strftime("%s%f")
 
```

Comment:
Let's drop this empty line too to keep this scope closer together
```suggestion
```

Reply:
ok